packages/client/src/arpa_reporter/views/Subrecipients.vue

```diff
@@ -23,31 +29,35 @@
       </div>
 
       <template slot="table-row" slot-scope="props">
-        <span v-if="props.column.field === 'upload_id'">
-          <router-link :to="`/uploads/${props.row.upload_id}`">
-            {{ props.row.upload_id }}
-          </router-link>
-        </span>
-
-        <ul v-else-if="props.column.field === 'record'" class="list-group list-group-flush">
-          <template v-for="(value, col) in props.row.json">
-            <template v-if="col === 'Unique_Entity_Identifier__c' || col === 'EIN__c'">
+        <div :class="{strikethrough: props.row.is_archived && props.column.field !== 'edit'}">
+          <span v-if="props.column.field === 'upload_id'">
+            <router-link :to="`/uploads/${props.row.upload_id}`">
+              {{ props.row.upload_id }}
+            </router-link>
+          </span>
+
+          <ul v-else-if="props.column.field === 'record'" class="list-group list-group-flush">
+            <template v-for="(value, col) in props.row.json">
+              <template v-if="col === 'Unique_Entity_Identifier__c' || col === 'EIN__c'">
+              </template>
+
+              <li v-else class="list-group-item" :key="col">
+                <span class="font-weight-bold">{{ col }}: </span>
+                {{ value }}
+              </li>
             </template>
-
-            <li v-else class="list-group-item" :key="col">
-              <span class="font-weight-bold">{{ col }}: </span>
-              {{ value }}
-            </li>
-          </template>
-        </ul>
-
-        <span v-else-if="props.column.field === 'edit'">
-          <router-link tag="button" class="btn btn-sm btn-secondary" :to="`/subrecipients/${props.row.id}`">Edit</router-link>
-        </span>
-
-        <span v-else>
-          {{props.formattedRow[props.column.field]}}
-        </span>
+          </ul>
+
+          <span class="d-flex flex-column" v-else-if="props.column.field === 'edit'" >
+            <router-link v-if="!props.row.is_archived" tag="button" class="btn btn-sm btn-secondary mb-2" :to="`/subrecipients/${props.row.id}`">Edit</router-link>
+            <button v-if="props.row.is_archived" class="btn btn-sm btn-primary" @click="archiveSubrecipient(props.row.id)">Restore</button>
```

Comment:
Having one function can be fine, but can we update the naming? Maybe something like `toggleArchiveSubrecipient`. It was confusing to read at first when the `archive` function can both archive and restore. Thoughts?

Reply:
Yeah, good note.  I definitely cut a corner there.  I do like the naming suggestion.